util/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mycelium-util"
-version = "0.1.0"
+version = "0.0.1"
```

Comment:
Why the version drop? Doesn't really matter, but it seems reasonable to leave it at `0.1.0`.

Reply:
The kernel prints the version on boot now (taken from the main package cargo version) and i thought it seemed pretty fake to print 0.1. So, I did a find and replace for all the version numbers. I can change it back if you prefer.